app/serializers/api/v1/participant_declaration_serializer.rb

```diff
@@ -34,6 +34,12 @@ class ParticipantDeclarationSerializer
       attribute :state do |declaration|
         declaration.current_state.dasherize
       end
+
+      attribute :has_passed, if: -> { FeatureFlag.active?(:participant_outcomes_feature) } do |declaration|
+        if declaration.npq?
+          declaration.outcomes.latest&.passed?
```

Comment:
ah I forgot about voided outcomes: if an outcome is voided we will currently return `false`, should we return `nil` instead?

Reply:
@cwrw 
Updated code to return nil if voided